app-admin/flannel-wrapper/files/flannel.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=flannel - Network fabric for containers (System Application Container)
+Documentation=https://github.com/coreos/flannel
+After=etcd.service etcd2.service etcd-member.service
+Before=docker.service
+
+[Service]
+Type=notify
+Restart=always
+RestartSec=10
+LimitNOFILE=40000
+LimitNPROC=1048576
+TimeoutStartSec=60
+ExecStartPre=/sbin/modprobe ip_tables
+ExecStartPre=-/usr/bin/rkt rm --uuid-file=/var/lib/coreos/flannel-wrapper1.uuid
+ExecStartPre=-/usr/bin/rkt rm --uuid-file=/var/lib/coreos/flannel-wrapper2.uuid
+ExecStartPre=/usr/bin/mkdir -p /var/lib/coreos
+Environment=FLANNEL_TAG=0.5.5
```

Comment:
Except they are versions that are compatible with gentoo because we try to stick to semver as much as possible for our projects. But doesn't really matter so it can stay as is.